packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -350,9 +367,38 @@ augmentFieldsDict(Posts, {
       },
     },
   },
+
+  glossary: {
+    resolveAs: {
+      type: GraphQLJSON,
```

Comment:
As mentioned elsewhere, this can be handled by defining a custom graphql type (via `addGraphQLSchema`) and then also assigning a corresponding simple-schema to the top-level `type` on the field defined in the schema file so the type generation gets handled correctly